app/views/medications/_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= form_for(@medication) do |f| %>
 <% if @medication.errors.any? %>
-  <div class="error_explanation">
-    <%= t('.error_explanation') %>
-  </div>
+<div class="error_explanation">
```

Comment:
Why remove the whitespace here? It seems like it's harder to understand that the div is only populated conditionally when it's at the same level as the `if` block.